preposterous-kumquat/preposterous-kumquat
# Implement Database w/ Postgres and Sequelize

#### What's this PR do?

This adds the Database with tables for Photos, Users, Keywords and Themes according to the schema. I used Postgres and Sequelize. I also added a route for testing. 
#### Where should the reviewer start?

Most new files reside in the /db directory. In the /server directory, I added a test route and a helper.js file for any functions the web-server needs to direct traffic.
#### How should this be manually tested?

You need to have postgres installed on your laptop. Check out http://postgresapp.com/

Open the postgres server by typing "psql" and then write "create database app;"

You'll also need to npm install... and also run "npm install sequelize -g"

Then, before starting node, type "sequelize db:migrate --config db/migrations/config/config.js

That will get you're db up and running. I'll create a gulp file or something to make that easier on us later.
#### Any background context you want to provide?

Sequelize is stooopid and the docs suck
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
